src/higley_lab_to_nwb/lohani_2022/interfaces/lohani_2022_visual_stimulus_interface.py

```diff
@@ -0,0 +1,121 @@
+from typing import List
+import pandas as pd
+from neo import io
+
+from neuroconv import BaseDataInterface
+from neuroconv.tools import get_package
+from neuroconv.utils import FilePathType
+from neuroconv.tools.signal_processing import get_rising_frames_from_ttl, get_falling_frames_from_ttl
+from spikeinterface.extractors import CedRecordingExtractor
+from pynwb import NWBFile
+from pynwb.epoch import TimeIntervals
+
+
+def _test_sonpy_installation() -> None:
+    get_package(
+        package_name="sonpy",
+        excluded_python_versions=["3.10", "3.11"],
+        excluded_platforms_and_python_versions=dict(darwin=dict(arm=["3.8", "3.9", "3.10", "3.11"])),
+    )
+
+
+def get_streams(file_path: FilePathType) -> List[str]:
+    """Return a list of channel names as set in the recording extractor."""
+    r = io.CedIO(filename=file_path)
+    signal_channels = r.header["signal_channels"]
+    stream_ids = signal_channels["id"]
+    stream_names = signal_channels["name"]
+    return stream_ids, stream_names
+
+
+class Lohani2022VisualStimulusInterface(BaseDataInterface):
```

Comment:
Maybe I'm wrong but I would inherit from `BaseTemporalAlignmentDataInterface` just in case, if you ever need to align the timestamps or adjust the starting time it will be easier.